components/prombench/apps/load-generator/main.py

```diff
@@ -36,13 +36,13 @@ def __init__(self, cfg, url, req_kwargs):
 
     def run(self):
         while True:
-            time.sleep(self.interval)
-
-            print("scaling deployment %s" % self.deployment)
-
+            print("scaling deployment %s down" % self.deployment)
```

Comment:
that would read better as 
print("scaling deployment %s to min" % self.deployment)
print("scaling deployment %s to max" % self.deployment)